budgetportal/tests/test_pages.py

```diff
@@ -228,7 +228,7 @@ def test_dataset_category_page(self):
 
         self.assertContains(response, "<title>test-slug - vulekamali</title>")
         self.assertContains(response, "basic-test-description")
-        self.assertContains(response, "Learn more about test-slug")
+        # self.assertContains(response, "Learn more about test-slug")
```

Comment:
I checked this and this test failure is not probably related to my code changes - please take a look. based on dicts we have hardcoded there is no way to find test-slug there. `guide` variable will be None in such case.

Reply:
Actually didn't try rebase - will check it out at the end

Reply:
Confirmed, that test is buggy also on muster
https://github.com/vulekamali/datamanager/pull/262

Reply:
thanks - fixed master and merged.